### PR TITLE
frontpage wireframe

### PR DESCRIPTION
**Pull request by mwernimont**
@jiwalker-usgs Heres a rough template of the frontpage

Don't pay too much attention to the outlines of the boxes and how they look a little gross, was just giving a sense of what the layout will be, not the look and feel.

Will be easier to do look and feel when we have content to fill the boxes.

### CODE REVIEW

File: viz.yaml [hunk 1]
```diff
@@ -46,6 +46,11 @@ process:
 visualize:
 publish:
   -
+    id: openSans
```
Comment (jiwalker-usgs) [inline viz.yaml]: 👍

File: layout/templates/frontPage.mustache [hunk 2]
```diff
@@ -1 +1,52 @@
-Hello World
\ No newline at end of file
+<div class="container">
+  <div id="topHalf">
+    <div id="sessions" class="topicHeader flexContainer box">
+      <h2>Sessions</h2>
+      <div id="users" class="box flex1">
+        new & returning users bar
```
Comment (jiwalker-usgs) [inline layout/templates/frontPage.mustache]: This should be &amp;

Comment (jiwalker-usgs) [inline layout/templates/frontPage.mustache]: though is this just filler?

Comment (mwernimont) [inline layout/templates/frontPage.mustache]: It was a shortcut, that text won't even be there in the future.

**Pull request merged**